src/madhuraMJun20/scannerClass/ReverseInt.java

```diff
@@ -0,0 +1,20 @@
+/*WAP to revers number (Consider Input as Integer not as String).*/
+
+package madhuraMJun20.scannerClass;
+
+public class ReverseInt {
+
+	void findReverse(int input) {
+		while (input > 0) {
+			int num1 = input % 10;
+			input = input / 10;
+			System.out.print(num1);
+		}
```

Comment:
But, here integer is not reversed exactly - it is chopped digit by digit and printed in reverse order.
Please implement integer reverse logic - which is just a small extension to your login.

void findReverseNumber(int number) {
		int reverseNum = 0;
		int mod =0;
		while(number>0) {
			mod = number%10;
			number = number/10;
		    reverseNum = mod + reverseNum*10 ;
		}
		System.out.println("Reverse Number is "+reverseNum);
	}

Reply:
Ok thanks. Updated the code as per suggestion.